addons/udes_stock/models/stock_location.py

```diff
@@ -588,6 +593,12 @@ def _process_inventory_adjustments(self, adjustments_request):
                     "prod_lot_id": stock_info.lot_id,
                 }
             )
+            inventory_line_ids.append(inventory_line.id)
+        inventory_data = self.get_inventory_adjustment_data()
+        inventory_data.update({
```

Comment:
`inventory_data["line_ids"] = [(6, 0, inventory_line_ids)]` is faster than {}.update()